Lib/test/test_support.py

```diff
@@ -691,6 +699,14 @@ def test_print_warning(self):
         self.check_print_warning("a\nb",
                                  'Warning -- a\nWarning -- b\n')
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_has_strftime_extensions(self):
+        if support.is_emscripten or sys.platform == "win32":
+            self.assertFalse(support.has_strftime_extensions)
+        else:
+            self.assertTrue(support.has_strftime_extensions)
```

Comment:
`support.has_strftime_extensions` will always be `False` when `sys.platform == "win32"`.

I suggest using the os-dependent version:

```python
def test_whatever(self):
    ...

# TODO: RUSTPYTHON
if sys.platform != "win32":
    # TODO: RUSTPYTHON
    test_whatever = unittest.expectedFailure(test_whatever)
```